consensus/enclave/mock/src/lib.rs

```diff
@@ -74,6 +78,74 @@ impl ConsensusServiceMockEnclave {
             output_public_keys,
         }
     }
+
+    /// Given a list of well formed encrypted txs + proofs and a root membership
+    /// element, decrypt and validate "original" Tx transactions, and if
+    /// successful return the list of transactions.
+    fn get_txs_from_inputs(
```

Comment:
This mostly moved from the bottom of the file, and attempts to mimic the real enclave implementation (sans the encryption, which the mock enclave does not do).